tools-in-path/test.sh

```diff
@@ -19,10 +19,13 @@ else
     exit 1
 fi
 
+version=$(dotnet --version)
```

Comment:
Can you replace everything including this version variable up to the `fi` with these two lines:
```
IFS='.-' read -ra VERSION_SPLIT <<< "$version"
dotnet tool update -g dotnet-ef --version "${VERSION_SPLIT[0]}.*-*"
```

If the `|| true` is needed, it would be nice if we add a comment that explains why.

Reply:
Note: `$1` is set by the test runner to the runtime version.

Reply:
> Note: `$1` is set by the test runner to the runtime version.

Thank you, I wasn't aware of this.